snippets/standard/io/io-exceptions/cs/io-exceptions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+class Program
+{
+    static void Main()
+    {
+	    var sw = OpenStream(@".\textfile.txt");
+        if (sw is null)
+            return;
+        sw.WriteLine("This is the first line.");
+        sw.WriteLine("This is the second line.");
+        sw.Close();
+    }
+
+   static StreamWriter OpenStream(string path)
+   {
+        if (path is null) {
+            Console.WriteLine("You did not supply a file path.");
```

Comment:
Do you want to return here? As written, it will throw an exception later...

Reply:
Yes, @BillWagner, I did want to return. Thanks for pointing that out.